data/wanderer/wanderers start.txt

```diff
@@ -260,113 +262,67 @@ mission "Wanderers: Translation Machine"
 			`	They respond, and Sayari translates, "They say that their species has encountered many other species in the past, and always has found those encounters to be valuable. Although..." She pauses to listen as the Wanderers continue speaking. "Although, they say that in their travels, far more often than they encounter living species, they instead only find themselves sifting through the refuse of a species that has destroyed itself through war or carelessness. They hope that you are not such a species, because..." Again she pauses as the Wanderers speak, then says, "Because, they hate to make new friends only to lose them again."`
 			
 			label end
-			`	Speaking through Sayari, you try to reassure the Wanderers that you have peaceful intentions. They speak with her for a while longer. She tells you, "They say that I will be allowed to stay here as an ambassador, and they will also let you land on most of their worlds. But they will not welcome any other Hai into their territory. I fear that they distrust the Hai because of the actions of the Unfettered, and being associated with me may harm your chances to win their favor. For your sake, we should find a different way for you to communicate with them."`
+			`	Speaking through Sayari, you try to reassure the Wanderers that you have peaceful intentions. They speak with her for a while longer. She tells you, "They say that I will be allowed to stay here as an ambassador, and they will also let you land on most of their worlds. But they will not welcome any other Hai into their territory. I fear that they distrust the Hai because of the actions of the Unfettered, and being associated with me may harm your chances to win their favor.`
+			`	"I'll have to remain here, so you'll need another way to communicate with them..." She pauses to talk to the Wanderers. "They're willing to create a translation device so you can communicate without an intermediary, but they need a large amount of data to train their algorithms. Human literature, speeches, interactive programs; anything with language and context."`
 			choice
-				`	"How can we do that?"`
-			`	She says, "There are scientists on the Hai world of <planet> who develop speaking machines, to allow the Hai who do not know the human language to communicate with humans. I will write a letter asking them to give you one of these speaking machines, so that you may talk with the Wanderers without needing me as an intermediary."`
+				`	"Where can I find the data?"`
+			`	"Your people have a large library at <planet>. You should fly there and find something suitable."`
```

Comment:
This line could be expanded on a bit, perhaps with some aspects similar to Eruk's description from the current version.
Currently, Eruk mentions that he heard of it from some of the Humans in Hai space, and also mentions that the station is home to 'a database of human literature and history'. I think it would be good to include such details, rather than just calling it 'a large library'.

Reply:
Agreed. It is disappointingly small, and I will borrow from Eruk's text to lengthen it.

Reply:
I have adapted Eruk's words to Sayari's mouth. How do you like it now?